src/hammer-vlsi/generate_properties.py

```diff
@@ -128,16 +143,11 @@ def main(args):
         outputs=[]
     )
 
-    output = []
-    output.extend(generate_interface(HammerSynthesisTool))
-    output.append("")
-    output.extend(generate_interface(HammerPlaceAndRouteTool))
-    output.append("")
-    output.extend(generate_interface(HammerDRCTool))
-    output.append("")
-    output.extend(generate_interface(HammerLVSTool))
-    print("\n".join(output))
- 
+    generate_interface(HammerSynthesisTool)
```

Comment:
Can we maybe make updating the implementation filename a flag, perhaps one that's enabled by default? I think it will help with testing/determinism if we can just print to stdout as well.

Reply:
yes, good idea.

Reply:
Fixed